<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>414</width>
    <height>435</height>
   </rect>
  </property>
  <widget class="QWidget" name="centralwidget">
   <property name="styleSheet">
    <string>background-color: #2C3E50; color: white;</string>
   </property>
   <widget class="QLabel" name="label">
    <property name="geometry">
     <rect>
      <x>100</x>
      <y>20</y>
      <width>200</width>
      <height>30</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string>font-size: 18px; font-weight: bold; color: #ECF0F1;</string>
    </property>
    <property name="text">
     <string>Password Manager</string>
    </property>
   </widget>
   <widget class="QLabel" name="label_2">
    <property name="geometry">
     <rect>
      <x>50</x>
      <y>80</y>
      <width>300</width>
      <height>20</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string>color: #BDC3C7;</string>
    </property>
    <property name="text">
     <string>Registering a new user will erase previous data</string>
    </property>
   </widget>
   <widget class="QLabel" name="label_3">
    <property name="geometry">
     <rect>
      <x>50</x>
      <y>130</y>
      <width>80</width>
      <height>20</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string>color: #ECF0F1;</string>
    </property>
    <property name="text">
     <string>Username:</string>
    </property>
   </widget>
   <widget class="QLineEdit" name="username_input">
    <property name="geometry">
     <rect>
      <x>140</x>
      <y>125</y>
      <width>200</width>
      <height>30</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">background-color: #ECF0F1; border-radius: 5px; padding: 5px;
color: black;
</string>
    </property>
   </widget>
   <widget class="QLabel" name="label_4">
    <property name="geometry">
     <rect>
      <x>50</x>
      <y>180</y>
      <width>80</width>
      <height>20</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string>color: #ECF0F1;</string>
    </property>
    <property name="text">
     <string>Password:</string>
    </property>
   </widget>
   <widget class="QLineEdit" name="password_input">
    <property name="geometry">
     <rect>
      <x>140</x>
      <y>175</y>
      <width>200</width>
      <height>30</height>
     </rect>
    </property>
    <property name="palette">
     <palette>
      <active>
       <colorrole role="WindowText">
        <brush brushstyle="SolidPattern">
         <color alpha="255">
          <red>0</red>
          <green>0</green>
          <blue>0</blue>
         </color>
        </brush>
       </colorrole>
       <colorrole role="Button">
        <brush brushstyle="SolidPattern">
         <color alpha="255">
          <red>236</red>
          <green>240</green>
          <blue>241</blue>
         </color>
        </brush>
       </colorrole>
       <colorrole role="Text">
        <brush brushstyle="SolidPattern">
         <color alpha="255">
          <red>0</red>
          <green>0</green>
          <blue>0</blue>
         </color>
        </brush>
       </colorrole>
       <colorrole role="ButtonText">
        <brush brushstyle="SolidPattern">
         <color alpha="255">
          <red>0</red>
          <green>0</green>
          <blue>0</blue>
         </color>
        </brush>
       </colorrole>
       <colorrole role="Base">
        <brush brushstyle="SolidPattern">
         <color alpha="255">
          <red>236</red>
          <green>240</green>
          <blue>241</blue>
         </color>
        </brush>
       </colorrole>
       <colorrole role="Window">
        <brush brushstyle="SolidPattern">
         <color alpha="255">
          <red>236</red>
          <green>240</green>
          <blue>241</blue>
         </color>
        </brush>
       </colorrole>
       <colorrole role="PlaceholderText">
        <brush brushstyle="SolidPattern">
         <color alpha="128">
          <red>0</red>
          <green>0</green>
          <blue>0</blue>
         </color>
        </brush>
       </colorrole>
      </active>
      <inactive>
       <colorrole role="WindowText">
        <brush brushstyle="SolidPattern">
         <color alpha="255">
          <red>0</red>
          <green>0</green>
          <blue>0</blue>
         </color>
        </brush>
       </colorrole>
       <colorrole role="Button">
        <brush brushstyle="SolidPattern">
         <color alpha="255">
          <red>236</red>
          <green>240</green>
          <blue>241</blue>
         </color>
        </brush>
       </colorrole>
       <colorrole role="Text">
        <brush brushstyle="SolidPattern">
         <color alpha="255">
          <red>0</red>
          <green>0</green>
          <blue>0</blue>
         </color>
        </brush>
       </colorrole>
       <colorrole role="ButtonText">
        <brush brushstyle="SolidPattern">
         <color alpha="255">
          <red>0</red>
          <green>0</green>
          <blue>0</blue>
         </color>
        </brush>
       </colorrole>
       <colorrole role="Base">
        <brush brushstyle="SolidPattern">
         <color alpha="255">
          <red>236</red>
          <green>240</green>
          <blue>241</blue>
         </color>
        </brush>
       </colorrole>
       <colorrole role="Window">
        <brush brushstyle="SolidPattern">
         <color alpha="255">
          <red>236</red>
          <green>240</green>
          <blue>241</blue>
         </color>
        </brush>
       </colorrole>
       <colorrole role="PlaceholderText">
        <brush brushstyle="SolidPattern">
         <color alpha="128">
          <red>0</red>
          <green>0</green>
          <blue>0</blue>
         </color>
        </brush>
       </colorrole>
      </inactive>
      <disabled>
       <colorrole role="WindowText">
        <brush brushstyle="SolidPattern">
         <color alpha="255">
          <red>0</red>
          <green>0</green>
          <blue>0</blue>
         </color>
        </brush>
       </colorrole>
       <colorrole role="Button">
        <brush brushstyle="SolidPattern">
         <color alpha="255">
          <red>236</red>
          <green>240</green>
          <blue>241</blue>
         </color>
        </brush>
       </colorrole>
       <colorrole role="Text">
        <brush brushstyle="SolidPattern">
         <color alpha="255">
          <red>0</red>
          <green>0</green>
          <blue>0</blue>
         </color>
        </brush>
       </colorrole>
       <colorrole role="ButtonText">
        <brush brushstyle="SolidPattern">
         <color alpha="255">
          <red>0</red>
          <green>0</green>
          <blue>0</blue>
         </color>
        </brush>
       </colorrole>
       <colorrole role="Base">
        <brush brushstyle="SolidPattern">
         <color alpha="255">
          <red>236</red>
          <green>240</green>
          <blue>241</blue>
         </color>
        </brush>
       </colorrole>
       <colorrole role="Window">
        <brush brushstyle="SolidPattern">
         <color alpha="255">
          <red>236</red>
          <green>240</green>
          <blue>241</blue>
         </color>
        </brush>
       </colorrole>
       <colorrole role="PlaceholderText">
        <brush brushstyle="SolidPattern">
         <color alpha="128">
          <red>0</red>
          <green>0</green>
          <blue>0</blue>
         </color>
        </brush>
       </colorrole>
      </disabled>
     </palette>
    </property>
    <property name="font">
     <font>
      <family>Sitka</family>
     </font>
    </property>
    <property name="styleSheet">
     <string notr="true">background-color: #ECF0F1; border-radius: 5px; padding: 5px;
color: black;
</string>
    </property>
    <property name="echoMode">
     <enum>QLineEdit::Password</enum>
    </property>
   </widget>
   <widget class="QPushButton" name="login_button">
    <property name="geometry">
     <rect>
      <x>140</x>
      <y>230</y>
      <width>100</width>
      <height>40</height>
     </rect>
    </property>
    <property name="cursor">
     <cursorShape>PointingHandCursor</cursorShape>
    </property>
    <property name="styleSheet">
     <string notr="true">background-color: #3498DB; color: white; border-radius: 10px; padding: 10px;
QPushButton {
    background-color: #0078D7;  /* Default blue */
    color: white;
    border-radius: 10px;
    padding: 10px 18px;
    font-size: 16px;
    font-weight: bold;
    border: 2px solid #005BB5;
    transition: 0.3s;
}

QPushButton:hover {
    background-color: #3399FF;  /* Brighter blue */
    border: 2px solid #ffffff;  /* White border for contrast */
    box-shadow: 0px 0px 20px 6px rgba(51, 153, 255, 0.9);  /* Stronger glow */
}

QPushButton:pressed {
    background-color: #0055A4;  /* Darker blue when clicked */
    box-shadow: 0px 0px 10px 4px rgba(51, 153, 255, 0.7);
}
</string>
    </property>
    <property name="text">
     <string>Login</string>
    </property>
   </widget>
   <widget class="QPushButton" name="register_button">
    <property name="geometry">
     <rect>
      <x>140</x>
      <y>290</y>
      <width>100</width>
      <height>40</height>
     </rect>
    </property>
    <property name="cursor">
     <cursorShape>PointingHandCursor</cursorShape>
    </property>
    <property name="styleSheet">
     <string notr="true">background-color: #1ABC9C; color: white; border-radius: 10px; padding: 10px;
QPushButton {
    background-color: #0078D7;  /* Default blue */
    color: white;
    border-radius: 10px;
    padding: 10px 18px;
    font-size: 16px;
    font-weight: bold;
    border: 2px solid #005BB5;
    transition: 0.3s;
}

QPushButton:hover {
    background-color: #3399FF;  /* Brighter blue */
    border: 2px solid #ffffff;  /* White border for contrast */
    box-shadow: 0px 0px 20px 6px rgba(51, 153, 255, 0.9);  /* Stronger glow */
}

QPushButton:pressed {
    background-color: #0055A4;  /* Darker blue when clicked */
    box-shadow: 0px 0px 10px 4px rgba(51, 153, 255, 0.7);
}
</string>
    </property>
    <property name="text">
     <string>Register</string>
    </property>
   </widget>
   <widget class="QLabel" name="label_5">
    <property name="geometry">
     <rect>
      <x>50</x>
      <y>360</y>
      <width>300</width>
      <height>20</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string>color: #E74C3C;</string>
    </property>
    <property name="text">
     <string>Don't forget your master password!</string>
    </property>
   </widget>
   <widget class="QPushButton" name="eyeIcon">
    <property name="geometry">
     <rect>
      <x>260</x>
      <y>180</y>
      <width>81</width>
      <height>21</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">background-color: #ECF0F1; border-radius: 5px; padding: 5px;
color: black;</string>
    </property>
    <property name="text">
     <string>👁 Show</string>
    </property>
   </widget>
  </widget>
  <widget class="QMenuBar" name="menubar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>414</width>
     <height>22</height>
    </rect>
   </property>
  </widget>
  <widget class="QStatusBar" name="statusbar"/>
 </widget>
 <resources/>
 <connections/>
</ui>
